kysely.ts

```diff
@@ -31,10 +33,12 @@ interface NewClientOpts {
 export const KyselyStore = <Session>(opts: NewClientOpts): SessionStore<Session> => {
 	// this assertion is a hack to make the Database type work
 	const table = (opts.table ?? defaults.table) as "telegraf-sessions";
+	const schema = (opts.schema ?? defaults.schema) as "public";
 
 	const client: Kysely<Database> = new Kysely(opts.config);
 
 	const create = client.schema
+		.withSchema(schema)
```

Comment:
Wouldn't this error when used with anything other than Postgres? Perhaps instead of defaulting to `"public"`, we can call `withSchema` only if schema is explicitly passed. Then we don't need to worry about Pg/others.

```suggestion
	const create = (schema ? client.withSchema(schema) : client.schema)
```